internal/services/compute/virtual_machine_scale_set.go

```diff
@@ -2345,3 +2266,22 @@ func flattenVirtualMachineScaleSetExtensions(input *compute.VirtualMachineScaleS
 	}
 	return result, nil
 }
+
+func flattenOrchestratedVirtualMachineScaleSetIdentity(input *identity.SystemAndUserAssignedMap) (*[]interface{}, error) {
```

Comment:
rather than doing this could we update the resource to support the System And User Assigned Identities instead?

Reply:
This comment was left in the schema from 2022 
https://github.com/hashicorp/terraform-provider-azurerm/blob/fd5f0433ab78d101979cb76c5ba8ba705df2b4cd/internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go#L156-L157
I just tested locally and it's still not supported
```
Virtual Machine Scale Set Name: "acctestOVMSS-240412080430308979"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: InvalidParameter: The value 'SystemAssigned' of parameter 'identity' is not allowed. Allowed values are: UserAssigned, None.
```